docs/components/breadcrumb/README.md

```diff
@@ -308,39 +308,6 @@ Use ```truncationEnabled``` to disable the truncation functionality.  Below show
 />
 ```
 
-Use the ```truncationThreshold``` prop to alter when truncation occurs:
-
-- Value must be a number between 0 and 1
-- Truncation occurs when (breadcrumb width) / (container width) exceeds the ```truncationThreshold``` value
-- The default value is 0.80 ( 80% )
-
-The below image and example code shows using ```truncationThreshold``` to set truncation to occur at 50%.
-
-<cdr-img class="cdr-doc-article-img" alt="Breadcrumb with truncation threshold at 50% and 80%" :src="$withBase(`/breadcrumb/Spec_API___Breadcrumb_Truncated_Threshold_50to80_16-4.png`)" />
```

Comment:
@KrisKnabel can you also delete these two files from the assets?

Reply:
Yes